packages/spatial/src/input/systems/ClientInputSystem.tsx

```diff
@@ -106,9 +107,16 @@ const execute = () => {
     getMutableComponent(eid, InputComponent).inputSources.set([])
   }
 
+  const stalePointers: Entity[] = []
+
   // update 2D screen-based (driven by pointer api) input sources
   for (const eid of pointersQuery()) {
     const pointer = getComponent(eid, InputPointerComponent)
+    // check if pointer camera entity still exists
```

Comment:
This PR will be merged soon:
https://github.com/ir-engine/ir-engine/pull/26
These new `stalePointers` blocks of code are not tested, but probably should have specific cases created for them :writing_hand:

Reply:
I will update #26 then, since this was merged first into dev